### DISCUSSION TITLE
Lauch wizard from another wizard

### DISCUSSION THREAD

**Discussion by emmexx** [filamentphp/filament#8847] [Help]
I want to guide user entering info for 2 related tables by using 2 wizards.
In the 1st wizard the user enters data for the 1st table, then,  upon saving, the 2nd wizard should start.
I need to pass to the 2nd wizard info about the 1st table record just created, at least the id.

At the moment I solved the problem with 

```
protected function getRedirectUrl(): string
    { 
        $record = $this->getRecord();
        $parameters = array('table1_id' => $record->getAttribute('id'));
        
        return \App\Filament\Resources\MembershipResource::getUrl(name: 'create', parameters: $parameters); 
    }
```
But I don't like to pass ids in clear. And to use redirect. Is there a proper filament/livewire way to accomplish that?
I don't want to use the relationship manager or modals.

Second question, related to the 1st one:
If the user stops entering data for the 2nd form there should be a way to start the wizard from the 1st Model table. I added an action column but I had to add a query parameter to pass to the 2nd wizard info about the "parent" record. Is it possible to do it some other way, without adding the id in the url?

I don't want to implement security by obscurity, I'm adding permission checks.

**Comment by pboivin** [comment]
> But I don't like to pass ids in clear.

Can you instead put the ID in the user's session?

> And to use redirect.

Personally, I don't see any issue with a redirect... Are you trying to avoid a page refresh?

**Reply by emmexx** [reply]
I was trying to keep it simple with a Filament solution that probably doesn't exist, using user session would only complicate my code.

A redirect is not the 1st thing that comes to my mind thinking about 2 related wizards but it's my problem.
